zebra-script/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-zcash_script = "0.1.6-alpha.0"
+zcash_script = { git = "https://github.com/jvff/zcash_script.git", rev = "d2680fa30" }
```

Comment:
This should be changed before merging. Either a new release of `zcash_script` should be made, or it should be changed to point to a commit in the `ZcashFoundation/zcash_script` repository.

Reply:
Since we're trying to get this merged in the next 24 hours, please just use a git commit hash here.